sdk/core/azure-core/inc/azure.hpp

```diff
@@ -1,9 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#pragma once
+#ifndef _az_CORE_AZURE_HPP
+#define _az_CORE_AZURE_HPP
```

Comment:
I think we should instead use `#pragma once` for consistency.

Reply:
If that's the guideline then for sure I will change it.
My problem with `#pragme once` is that they are (de jure) non-standard, plus theoretically you can run into a problem when you use filesystem links and the same file is available via many paths that all lead to the same file location.

Reply:
+1 for `#pragma once`. Although it's not part of standard, but it works on all known compilers.

Reply:
I am going to use it for consistency, I am fine with it; I'll just make sure we finalize on it during our discussions meeting.